src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiBuildInfo.java

```diff
@@ -352,6 +365,91 @@ public void setVolumeName( String volumeName )
         this.volumeName = volumeName;
     }
 
+    /**
+     * Returns extra metadata of the build, mapped as a XML-RPC struct.
+     * @return Map of XML-RPC types of extra data.
+     * @see <a href="https://ws.apache.org/xmlrpc/types.html">https://ws.apache.org/xmlrpc/types.html</a>
+     */
+    public Map<String, Object> getExtra()
+    {
+        return extra;
+    }
+
+    /**
+     * Sets the extra metadata for the build.
+     * @param extra The extra metadata, mapped to XML-RPC types.
+     * @see #getExtra()
+     */
+    public void setExtra( Map<String, Object> extra )
+    {
+        this.extra = extra;
+    }
+
+    @Override
+    public void writeExternal( ObjectOutput out )
+            throws IOException
+    {
+        out.writeInt( VERSION );
+        out.writeInt( id );
+        out.writeInt( packageId );
+        ExternalizableUtils.writeUTF( out, name );
+        ExternalizableUtils.writeUTF( out, version );
+        ExternalizableUtils.writeUTF( out, release );
+        out.writeObject( completionTime );
+        out.writeObject( creationTime );
+        ExternalizableUtils.writeUTF( out, nvr );
+        out.writeObject( taskId );
+        out.writeObject( ownerId );
+        ExternalizableUtils.writeUTF( out, ownerName );
+        out.writeObject( buildState );
+        out.writeObject( creationEventId );
+        ExternalizableUtils.writeUTF( out, mavenGroupId );
+        ExternalizableUtils.writeUTF( out, mavenArtifactId );
+        ExternalizableUtils.writeUTF( out, mavenVersion );
+        ExternalizableUtils.writeUTF( out, platform );
+        out.writeObject( extra );
+        ExternalizableUtils.writeUTF( out, source );
+        // XXX: This has @JsonIgnore
+        out.writeObject( typeNames );
```

Comment:
This has @JsonIgnore because it's not in the data model, but it's impossible to recreate this field without going online.